script/Deploy.s.sol

```diff
@@ -11,24 +11,12 @@ import {CreatorTokenSwapRouter} from "src/CreatorTokenSwapRouter.sol";
 
 contract Deploy is Script {
```

Comment:
Rather than modifying the existing script, I'd recommend just creating a new script file with a different name (`DeployOpenFactory.s.sol`) or similar.